.github/workflows/use-latest-docker-image-for-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Use latest docker image for CI
+
+on:
+    #pull_request:
+    workflow_dispatch:
+  
+concurrency:
+    group: ${{ github.ref }}
+    cancel-in-progress: true
+
+env:
+  ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
+  MAIN_PYTHON_VERSION: '3.10'
+  FLUENT_IMAGE_TAG: v25.2.0
+  FLUENT_VERSION: 252
+
+jobs:
+  test:
+    name: Update latest Fluent image
+    runs-on: [self-hosted, pyfluent]
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cleanup previous docker containers
+        if: always()
+        run: make cleanup-previous-docker-containers
+
+      - name: Update Fluent image
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+        if: github.ref == 'refs/heads/main'
+        run: |
+          DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/ansys/pyfluent:${{ env.FLUENT_IMAGE_TAG }} | sed 's/.*@//')
```

Comment:
Do we need to first pull the latest docker image locally before calling `docker inspect`?

Reply:
@hpohekar Sorry if I'm missing something, but I don't seem to find the docker pull step in the latest version of the CI script.

Reply:
@mkundu1 Done. Thank you.